cmd/util_fp_finder.go

```diff
@@ -28,40 +30,29 @@ func createFpFinderCommand() *cobra.Command {
 		}),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			fpFinder := util.NewFpFinder()
-
-			sortEnabled, err := cmd.Flags().GetBool("sort")
-			if err != nil {
-				logger.Error().Err(err).Msg("Failed to read value for 'sort' flag")
-				return err
-			}
-
-			uniqEnabled, err := cmd.Flags().GetBool("uniq")
-			if err != nil {
-				logger.Error().Err(err).Msg("Failed to read value for 'uniq' flag")
-				return err
-			}
-
 			filenameArg := args[0]
 			if !checkFilePath(filenameArg) {
 				return fmt.Errorf("file %s doesn't exist", filenameArg)
 			}
 
-			return fpFinder.FpFinder(filenameArg, sortEnabled, uniqEnabled)
+			if extendedDictPath != "" && !checkFilePath(extendedDictPath) {
+				return fmt.Errorf("extended dictionary %s doesn't exist", extendedDictPath)
+			}
+
+			return fpFinder.FpFinder(filenameArg, extendedDictPath, englishDictionaryCommitHash)
 		},
 	}
 
 }
 
 func buildFpFinderCommand() {
 	utilCmd.AddCommand(fpFinderCommand)
-	fpFinderCommand.Flags().BoolP("sort", "s", false, "Sort the output alphabetically")
-	fpFinderCommand.Flags().BoolP("uniq", "u", false, "Remove duplicated value from the output")
+	fpFinderCommand.Flags().StringVarP(&extendedDictPath, "extended-dictionary", "e", "", "Absolute or relative path to the extended dictionary")
+	fpFinderCommand.Flags().StringVarP(&englishDictionaryCommitHash, "english-dictionary-commit-hash", "c", "8179fe68775df3f553ef19520db065228e65d1d3", "English dictionary commit hash from Github")
 }
 
 func checkFilePath(path string) bool {
 	_, err := os.Stat(path)
-	if os.IsNotExist(err) {
-		return false
-	}
+	os.IsNotExist(err)
```

Comment:
Am I missing something? `os.IsNotExist(err)` will return a boolean that's not used, nothing else, right?